server/base/src/main/java/org/apache/accumulo/server/master/balancer/ChaoticLoadBalancer.java

```diff
@@ -163,14 +160,4 @@ public long balance(SortedMap<TServerInstance,TabletServerStatus> current, Set<K
 
     return 100;
   }
-
-  @Deprecated
-  @Override
-  public void init(ServerConfiguration conf) {
-    throw new NotImplementedException();
```

Comment:
This method has no impact at runtime inside the Accumulo master service. It would only affect somebody calling it directly... and that would probably only happen during testing. So, I *think* this exception would have been here to prevent testing bugs (accidentally calling the wrong init during testing). Might be best to leave it... though if our tests are correct, it should not matter.

Reply:
Yeah I can add it back.

Reply:
Thinking about it some more... the `UnsupportedOperationException` should really exist in the parent. Actually, I think we only kept this method around at all, because of API stability reasons... but this isn't public API, and I don't think it's very stable as is anyway. I would probably leave it as-is in previous release lines, and reconsider this whole API in 2.0 (or some other future point).